README.md

```diff
@@ -80,7 +80,6 @@ If all of those commands don't work, the following process might:
 ## Configuration
 Within Admin Settings you can set following Attributes
 
-- Force objects re-init (ReInitialize all Objects within ioBroker)
```

Comment:
Whats the reason for this?

Reply:
That function doesn't exist. Maybe it did in the past.